From ccace3858e0ed2d87c59dc3c7385fb8c765928b6 Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Sun, 4 May 2025 19:02:59 +0800
Subject: [PATCH] AOSCOS: Unpin setuptools_scm
X-Developer-Signature: v=1; a=openpgp-sha256; l=764; i=xtexchooser@duck.com;
 h=from:subject; bh=fFMWctMdQ0M3irrbTF4jrwuTCt85s60gcK3VoQ0hw/U=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDBniniciJzLF1Tc2N9zc3uW7Qymrb+YPmxI28di68t3a0
 hfmcH/rKGVhEONikBVTZCkybPBm1UnnF11WLgszh5UJZAgDF6cATES/jOG/z2yf8+vdjbdtOTJ7
 prj4GaO+qunhM3bVpp+5KM4vt3fhI0aG9zOfLn/LEfK0w19keanjpR8fnPwYJDcqcxtt/Z2nc9O
 MGQA=
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

The reason why it was pinned is that its generated
files are no longer compatible with Python 2.x.
However, we have dropped Python 2.x support.

Fixes: 9780d32aea9a ("Pin `setuptools_scm` to <8")
Signed-off-by: xtex <xtexchooser@duck.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index b609717607d7..67ad619fc965 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,7 +3,7 @@ requires = [
     "setuptools; python_version != '3.3'",
     "setuptools<40.0; python_version == '3.3'",
     "wheel",
-    "setuptools_scm<8.0"
+    "setuptools_scm"
 ]
 build-backend = "setuptools.build_meta"
 

base-commit: 1ae807774053c071acc9e7d3d27778fba0a7773e
-- 
2.49.0

